bigquery_etl/cli/query.py

```diff
@@ -784,7 +792,7 @@ def backfill(
             run_checks=checks,
         )
 
-        if not depends_on_past:
+        if not depends_on_past and parallelism > 0:
```

Comment:
`pytest` doesn't like Multiprocessing apparently. I get an indefinite hang when I try to run a test with parallelism on. This is just a way to force sequential runs in tests and otherwise.